package.json

```diff
@@ -37,6 +37,7 @@
     "@18f/identity-normalize-yaml": "^1.0.0",
     "@axe-core/puppeteer": "^4.5.1",
     "@babel/eslint-plugin": "^7.14.5",
+    "@babel/preset-react": "^7.22.5",
     "@types/jest": "^29.2.6",
```

Comment:
@allthesignals  devDependencies vs dependencies?  I feel like I should move this to the dependencies list. When we paired tho we installed it as devDependencies so did the same but now wondering.... thoughts?

Reply:
Now that you mention it, I think we should put this in `dependencies`, not `devDependencies`.

My thinking is that `dependencies` should contain the bare minimum necessary to build the live site, while `devDependencies` is for everything else (test dependencies, linting, etc).

I don't know that it makes much difference at the moment, but I'd like to see if there's a way we can make it so Cloud.gov Pages only installs production dependencies, so that the builds are faster. I would have thought it worked this way already since I checked and we do set `NODE_ENV=production`, which [should exclude `devDependencies`](https://docs.npmjs.com/cli/v8/commands/npm-install#:~:text=or%20when%20the%20NODE_ENV%20environment%20variable%20is%20set%20to%20production), but clearly the preview build is working just fine 🤷

Reply:
Thanks for input. I thought same- this is needed for the app where as devDependencies are more for development and testing- thing that are not needed at runtime. I moved package with commit 3fb297e4d364714abfc8e57575c2805c3f1de927